### DISCUSSION TITLE
标准在文献附录中是Z

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#134] [Q&A]
你好，标准在文献附录中，是Z标识，在国标中要求S，并且标准在最后有个standard，国标没有这要求。是我哪里搞错了吗？
![image](https://user-images.githubusercontent.com/45897471/166866084-e8d0c665-4bcb-426e-b70e-dd88e065a875.png)

**Comment by zepinglee** [comment]
需要给出相应的 bib 数据库。

**Reply by earthmoon** [reply]
你好，数据库我现在传上来
[Uploading refs.zip…]()

**Reply by earthmoon** [reply]
[refs.zip](https://github.com/whutug/whu-thesis/files/8630543/refs.zip)

**Reply by zepinglee** [reply]
其实直接把下面的内容贴出来最方便。

```bib
@misc{RN34,
   author = {环境保护部},
   title = {水质 总氮的测定 碱性过硫酸钾消解紫外分光光度法},
   publisher = {中国质检出版社},
   pages = {8},
   ISBN = {HJ 636 - 2012},
   year = {2012},
   type = {Standard}
}
```

参考 `texdoc biblatex-gb7714-2015` 文档 4.3.2 节，`@misc` 改成 `@book`，`type = {Standard}` 改成 `entrysubtype = {standard}`。

**Reply by earthmoon** [reply]
你好，好像只需要`type = {Standard}`改成`entrysubtype = {standard}`就可以了。不知道为什么
